malinka.el

```diff
@@ -730,7 +773,35 @@ EVENT is ignored."
       (with-temp-buffer
         (malinka--select-project build-dir)))))
 
+(defun malinka--handle-bear-finish (process event)
+  "Handle all events from the project bear command PROCESS.
+EVENT is ignored."
+  (when (memq (process-status process) '(signal exit))
+    (let* ((project-map       (process-get process 'malinka-project-map))
+           (project-name      (malinka--project-name project-map))
+           (build-dir         (malinka--project-build-directory project-map)))
+      (malinka--info "Bear command for \"%s\" finished. Proceeding to process the output" project-name)
+      ;; TODO Do we need to handle irony issue as in malinka--handle-cmake-finish?
```

Comment:
No I don't think we do. I have not really tested with latest versions of  irony to be honest. I should, but for my normal work I only use rtags.